e2e/tests/features/401-report-data.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from '@playwright/test';
```

Comment:
suggestion: the test numbers like `401` are to ensure execution order

In this case, the test does not start a new context, so I suggest we just continue the existing order and name it `211-report-data`